libphal/phal_sbe.C

```diff
@@ -242,6 +246,24 @@ sbeError_t captureFFDC(struct pdbg_target *proc)
 	if (status == (SBEFIFO_PRI_UNKNOWN_ERROR | SBEFIFO_SEC_HW_TIMEOUT)) {
 		log(level::ERROR, "SBE chipop timeout reported(%s)",
 		    pdbg_target_path(proc));
+		// only for primary processor SBE run p10_extract_sbe_rc hwp and
+		// perform recovery actions
+		if (isPrimaryProc(proc)) {
+			try {
+				sbeErrorHandler(proc);
```

Comment:
Is this path is tested ?
This hw execution won't work  for the chip-ops , which is called in default devtree mode. This procedure excution need kerenl back-end. "pdbg::init(PDBG_BACKEND_KERNEL)"